modules/rocksdb-common/src/main/java/org/apache/ignite/internal/rocksdb/ColumnFamily.java

```diff
@@ -45,21 +46,27 @@ public class ColumnFamily {
     /** Column family handle. */
     private final ColumnFamilyHandle cfHandle;
 
+    /** Column family options, if exclusive to this CF. */
+    @Nullable
+    private final ColumnFamilyOptions options;
```

Comment:
How about renaming the field to make it easier to understand that it's something exclusive to this object? Like `ownedOptions` or something similar

Reply:
I gave more thought about this and I think it's probably easier to name it either `reserved` or `private`.

Reply:
I like `private` and `exclusive`